cto_ai/sdk.py

```diff
@@ -82,6 +82,18 @@ def track(
         pass
 
 
+def start(
+    workflowName: str
+):
+    """Start a workflow from the sdk"""
+    try:
+        daemon_request.start_op(dict(trigger=True, workflowName=workflowName, tags=["trigger"]))
+
+    # It remains unclear what to do with these errors
+    except RequestException:
+        pass
```

Comment:
@lucamaraschi brought this up in another project: human readable errors and error codes. Maybe we start with a page in Notion and add an error code to it along with a rough idea of what went wrong and what is needed to debug?

Reply:
True that, ok let me come up with something.